rdrf/rdrf/services/io/notifications/longitudinal_followups.py

```diff
@@ -135,6 +148,11 @@ def send_longitudinal_followups(now):
 
         patient = patient_entries[0].patient
 
+        patient_registry = patient.rdrf_registry.first()
+
+        if not patient_registry.has_feature(RegistryFeatures.LONGITUDINAL_FOLLOWUPS):
```

Comment:
Is there a better way to do this? Is it worth having a button somewhere that just causes this process to exit completely? (rather than skip to the next iteration).

I don't have a good answer to this.

Reply:
There's no way to stop a running lambda, so I think something similar to this is the only way.

Reply:
I mean using `break` instead of `continue`, to exit the loop entirely.

Reply:
Yeah I think that works